docs/source/get-started/android/requirements.rst

```diff
@@ -0,0 +1,126 @@
+Install the required tools and library
+=======================================
+
+In this section, we will install the required libraries and tools to build and develop Android SIP
+client applications supporting voice/VoIP calls, video, and secure communication using TLS and
+secure RTP (SRTP).
+
+We tried to present the minimum and optimum features in this tutorial. You can install other,
+optional features by clicking the feature in :doc:`sip-sdk-features` page.
+
+
+.. contents:: Installation steps:
+   :depth: 2
+   :local:
+
+
+
+Install Android Android Studio and Android NDK
+------------------------------------------------------
+
+While PJSIP only requires the NDK to build, you usually need Android Studio to develop your
+application, and it is recommended to install the NDK from Android Studio's SDK manager anyway,
+hence let's install them by following the instructions here:
+
+  - Install `Android Studio <https://developer.android.com/studio>`__
+  - Install `Android NDK <https://developer.android.com/ndk>`__.
+
+This tutorial uses *Android Studio Ladybug Feature Drop 2024.2.2* and *NDK 28.0.12916984*
+on *Ubuntu 22.04*.
+
+
+Install SWIG
+-------------------------------------------
+
+SWIG is used to create high level language bindings (such as Java, C#) for
+:doc:`PJSUA2 API </api/pjsua2/index>`.
+
+To install it, follow the instructions from `SWIG <http://www.swig.org/download.html>`__ homepage.
+
+Make sure SWIG is accessible in the PATH. You can check by running:
+
+.. code-block:: shell
+
+  $ swig -version
+
+  SWIG Version 4.0.2
+
+  Compiled with g++ [x86_64-pc-linux-gnu]
+
+  Configured options: +pcre
+
+  Please see http://www.swig.org for reporting bugs and further information
+
+
+
+Download and extract Oboe
+-------------------------------------------
+
+We recommend Oboe for Android audio device:
+
+- download Oboe prefab package (`.aar` file) from https://github.com/google/oboe/releases
+- extract it somewhere in your system, save the location to **OBOE_DIR** environment variable
+  (we will refer to it later).
+
+  .. code-block:: shell
+
+    $ export OBOE_DIR=/home/whoever/Android/oboe-1.9.3
+
+For additional information, see :ref:`oboe`.
+
+
+.. _android_openssl:
+
+Install OpenSSL
+-------------------------------------------
+
+Let's use TLS in our demo as a good security practice. To install it:
+
+1. Download OpenSSL release from https://github.com/openssl/openssl/releases (we tested with
+   versions from 1.0.2s to 3.4)
```

Comment:
Considering that our included libsrtp has deprecated OpenSSL < 1.1.0, I think it's better to put 1.1.0 here.